MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -265,6 +271,11 @@ void EnggDiffractionPresenter::processShutDown() {
   cleanup();
 }
 
+void EnggDiffractionPresenter::processPlotRepChange() {
+  m_view->saveSettings();
```

Comment:
Do you really want to save the settings every time the user changes this? For all the other settings what happens currently is that they will be saved when the interface is closed, for the next session.